main.go

```diff
@@ -101,6 +102,23 @@ func petasos(arguments []string) int {
 
 	infoLog.Log("configurationFile", v.ConfigFileUsed())
 
+	u := v.Sub(tracingConfigKey)
```

Comment:
If the tracing config is not provided, I thought the plan is to default to notrace. 

Is that correct @johnabass ?

Reply:
This is related to the [discussion](https://github.com/xmidt-org/candlelight/pull/23#discussion_r608037321) in https://github.com/xmidt-org/candlelight/pull/23

Reply:
Hi @kcajmagic  This is handled in the latest commit. we are using `loadTracing` as of now.